### PR TITLE
Libinjector wait exit

### PR DESCRIPTION
**Pull request by skvl**
The patch adds method to exit injection thread after target process injection complete.

This is accomplished with:
* New functions in `libdrakvuf`:
  * `drakvuf_enum_threads` - returns a hash table with `ETHREAD` for a given `EPROCESS`.
  * `drakvuf_get_thread` - returns a `ETHREAD` for `PID:TID` if any.
* New `drakvuf` option `--exit-injection-thread` which enables `RtlExitUserThread` injection for injection thread.

### CODE REVIEW

File: src/libdrakvuf/os.c [hunk 1]
```diff
@@ -553,6 +553,31 @@ bool drakvuf_is_process_suspended(drakvuf_t drakvuf, addr_t process, bool* statu
...
+    return ret;
+}
+
+addr_t drakvuf_get_thread(drakvuf_t drakvuf, addr_t process, uint32_t tid)
+{
+    GHashTable* threads = drakvuf_enum_threads(drakvuf, process);
+    if (!threads)
+        return 0;
+
+    addr_t thread = (addr_t)g_hash_table_lookup(threads, (gconstpointer)(uint64_t)tid);
```
Comment (tklengyel) [inline src/libdrakvuf/os.c:576]: GUINT_TO_POINTER instead of casting.

**Comment by tklengyel** [comment]
For such a large PR I'll need a detailed PR description of what is being added/changed.

File: src/libdrakvuf/os.c (hunk 1 above)
Comment (skvl) [reply src/libdrakvuf/os.c]: Thanks!

**Comment by skvl** [comment]
> For such a large PR I'll need a detailed PR description of what is being added/changed.

Done.

**Comment by tklengyel** [comment]
@drakvuf-jenkins This is OK to test

**Comment by skvl** [comment]
@tklengyel , I have re-based the branch.

**Comment by skvl** [comment]
I have re-based the branch

**Pull request merged**